app/javascript/packs/styles/main.scss

```diff
@@ -1,8 +1,70 @@
 @import 'constants';
+@import '~react-bootstrap-table/dist/react-bootstrap-table.min';
 
 body {
   background-color: $color-concrete;
   color: $color-emperor;
   font-family: "Open Sans",Arial,Helvetica,sans-serif;
   font-size: 13px;
 }
+
+.react-bs-table {
```

Comment:
for styling please use styled-components library in here we will put global styles like default font sizes and bootstrap overrides
https://www.styled-components.com/